### ISSUE TITLE
Links open in Modal/Popup

### ISSUE THREAD

**Issue by AlexisSerneels** [laravel/nova-issues#4350]
Support requested elevated from this basic repository

**Comment by crynobone** [comment]
Provide reproducing code to this issue, see https://github.com/nova-issues as example.

**Comment by AlexisSerneels** [comment]
I really can't say much more than the video screenshot provided.
You click on a resource menu link, instead of changing the all page, it loads the content of the link within a modal.
That's it. No code involves.

Had this behavior for ages though. Even on V3.

**Comment by crynobone** [comment]
v3 doesn't use Inertia so that statement juat simply impossible. More reason a reproducing repository is required before we can do anything here.

**Comment by AlexisSerneels** [comment]
Oh man, I hate when paying support lands on your end and you have to prove it's the product fault, not yours.

Video shows the issue.

Close the issue if you want and let the bug be within your paying product.
This is not an open source project were I would never reply this way.

**Comment by crynobone** [comment]
@AlexisSerneels you are the only person so far who has reported this issue. We can't confirm the issue due to Nova code itself or 3rd party packages (which happen a lot). We also have extensive dusk tests running on every code change which hasn't replicated any of the above issue.

**Comment by AlexisSerneels** [comment]
Every new click injects the new page content within:

```<div style="position: fixed; width: 100vw; height: 100vh; padding: 50px; box-sizing: border-box; background-color: rgba(0, 0, 0, 0.6); z-index: 200000;"><iframe style="background-color: white; border-radius: 5px; width: 100%; height: 100%;"></iframe></div>```

Happens on FF, Chrome and Safari.

No addition Nova package is installed, resource is only using stock Nova fields.

Clicking on the resource link send a normal GET Request to /admin/resources/XXX

I can not share reproducing repository due to privacy reasons.

**Comment by JshGrn** [comment]
Your video shows its not a stock nova install with no packages, nobody can help you without being able to replicate it, your video is no better than a screenshot of the popup.

**Comment by JshGrn** [comment]
I am not part of the team if you are complaining about my response, I am just going through the issues to find something related to mine.

This doesn't happen to anyone else but you so your composer.json file posted would be of great help, you provided a video showing something you could have shown in a screenshot.

**Comment by crynobone** [comment]
You can send a request to nova@laravel.com

But again,

1. no one else has yet to report anything similar to what you are facing.
2. Already tried replicating the issue via https://github.com/nova-issues/issue-4350 based on reproducing step above without any sucess 
3. @JshGrn doesn't work at Laravel

**Comment by AlexisSerneels** [comment]
Support request elevated.
